db_setup.sql

```diff
@@ -0,0 +1,57 @@
+
+CREATE SEQUENCE transactions_seq
+  start 1
+  increment 1;
+
+CREATE SEQUENCE customers_seq
+  start 1
+  increment 1;
+ 
+CREATE SEQUENCE products_seq
+  start 1
+  increment 1;
+ 
+CREATE TABLE customers (
+	"id" int NOT null default nextval('customers_seq'),
```

Comment:
generally use uuid for primary keys instead of integers, but for the purpose of this project keep it